test/suites/chaos/suite_test.go

```diff
@@ -69,8 +69,8 @@ var _ = Describe("Chaos", func() {
 			defer cancel()
 
 			provider := awstest.AWSNodeTemplate(v1alpha1.AWSNodeTemplateSpec{AWS: v1alpha1.AWS{
-				SecurityGroupSelector: map[string]string{"karpenter.sh/discovery": settings.FromContext(env.Context).ClusterName},
-				SubnetSelector:        map[string]string{"karpenter.sh/discovery": settings.FromContext(env.Context).ClusterName},
+				SecurityGroupSelector: map[string]string{"karpenter.sh/discovery": lo.Must(os.LookupEnv("CLUSTER_NAME"))},
```

Comment:
Is there a way to lookup this value once in the environment and then refer to it that way rather than having to continually do an `os.LookupEnv`?

Reply:
A couple of options there:
1) We could introduce a function along the lines of `env.GetClusterName()`
2) We could do something similar to what was done before, inject `options` with `ClusterName` populated.
I'd lean toward the first option as it doesn't couple this to Karpenter's cli flags.